CODEOWNERS

```diff
@@ -43,3 +43,4 @@ homeassistant/components/light/tplink.py @rytilahti
 homeassistant/components/switch/tplink.py @rytilahti
 homeassistant/components/climate/eq3btsmart.py @rytilahti
 homeassistant/components/*/xiaomi_miio.py @rytilahti
+homeassistant/components/climate/ephember.py @ttroy50
```

Comment:
Please sync to HEAD - there were several changes to this file meanwhile.
Also add the new file in alphabetical place.